Packs/CommonScripts/Scripts/HelloDemisto/HelloDemisto.py

```diff
@@ -0,0 +1,7 @@
+from CommonServerPython import *  # noqa: F401
+
+
+if __name__ in ("__main__", "__builtin__", "builtins"):
```

Comment:
The script does not use a `main()` function, and the logic is not wrapped in a `try/except` block. All XSOAR scripts should encapsulate their logic within a `main()` function for better structure and maintainability, and the `main()` function should include a `try/except` block to catch exceptions and call `return_error()` for proper error handling.
```suggestion
def main():
    try:
        name = demisto.args().get('name') or 'NoBody'
        name = name.strip() if name else 'Demisto'
        return_results(f'Hello, {name}!')
    except Exception as e:
        return_error(f'Failed to execute HelloDemisto script. Error: {e}')

if __name__ in ('__main__', '__builtin__', 'builtins'):
    main()
```